web/src/admin/applications/wizard/auth-method-choice/ak-application-wizard-authentication-method-choice.choices.ts

```diff
@@ -25,167 +25,152 @@ type ModelConverter = (provider: OneOfProvider) => ModelRequest;
             html`<ak-application-wizard-authentication-by-saml-configuration></ak-application-wizard-authentication-by-saml-configuration>`,
-        ProviderModelEnum.SamlSamlprovider,
-        (provider: OneOfProvider) => ({
+        modelName: ProviderModelEnum.SamlSamlprovider,
+        converter: (provider: OneOfProvider) => ({
             providerModel: ProviderModelEnum.SamlSamlprovider,
             ...(provider as SAMLProviderRequest),
         }),
-    ],
-    [
-        "radiusprovider",
-        msg("RADIUS (Remote Authentication Dial-In User Service)"),
-        msg("Configure RADIUS provider manually"),
-        () =>
+        component: "",
+        iconUrl: "/static/authentik/sources/saml.png",
+    },
+    {
+        formName: "radiusprovider",
+        name: msg("RADIUS (Remote Authentication Dial-In User Service)"),
+        description: msg("Configure RADIUS provider manually"),
+        renderer: () =>
             html`<ak-application-wizard-authentication-by-radius></ak-application-wizard-authentication-by-radius>`,
-        ProviderModelEnum.RadiusRadiusprovider,
-        (provider: OneOfProvider) => ({
+        modelName: ProviderModelEnum.RadiusRadiusprovider,
+        converter: (provider: OneOfProvider) => ({
             providerModel: ProviderModelEnum.RadiusRadiusprovider,
             ...(provider as RadiusProviderRequest),
         }),
-    ],
-    [
-        "scimprovider",
-        msg("SCIM (System for Cross-domain Identity Management)"),
-        msg("Configure SCIM provider manually"),
-        () =>
+        component: "",
+        iconUrl: "/static/authentik/sources/radius.svg",
+    },
+    {
+        formName: "scimprovider",
+        name: msg("SCIM (System for Cross-domain Identity Management)"),
+        description: msg("Configure SCIM provider manually"),
+        renderer: () =>
             html`<ak-application-wizard-authentication-by-scim></ak-application-wizard-authentication-by-scim>`,
-        ProviderModelEnum.ScimScimprovider,
-        (provider: OneOfProvider) => ({
+        modelName: ProviderModelEnum.ScimScimprovider,
+        converter: (provider: OneOfProvider) => ({
             providerModel: ProviderModelEnum.ScimScimprovider,
             ...(provider as SCIMProviderRequest),
         }),
-    ],
-];
-
-function mapProviders([
-    formName,
-    name,
-    description,
-    _,
-    modelName,
-    converter,
-    note,
-]: ProviderType): LocalTypeCreate {
-    return {
-        formName,
-        name,
-        description,
         component: "",
-        modelName,
-        converter,
-        note,
-    };
-}
-
-export const providerModelsList = _providerModelsTable.map(mapProviders);
+        iconUrl: "/static/authentik/sources/scim.png",
+    },
+];
 
 export const providerRendererList = new Map<string, ProviderRenderer>(
-    _providerModelsTable.map(([modelName, _0, _1, renderer]) => [modelName, renderer]),
+    providerModelsList.map((tc) => [tc.formName, tc.renderer]),
```

Comment:
I'm glad this wasn't too confusing.  If data is uniform, I like tuples, and I like annotating them.